specification/src/chapter2.adoc

```diff
@@ -155,8 +156,59 @@ break security guarantees, either directly or indirectly. For example:
 
 | CAT_NNN
 | Invasive subsystems MUST be controlled, or moderated, by a RoT.
+
+|===
+
+==== Event counters
+
+Event counters are commonly used for performance management and resource allocation. 
+
+However, they can also pose a security risk. For example, one workload measuring an operation in a different workload, or an operation by higher privilege software, could be able to reveal assets used in those operations.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#
+| Requirement
+
+| CAT_NNN
+| Lower privilege software MUST NOT be able to measure higher privilege software.
+
+| CAT_NNN
+| Software in one domain MUST NOT be able to measure software in a different domain, without consent.
+
+|===
+
+==== Platform quality of service
+
+Server platforms can provide _platform quality of service (QoS)_ features, consisting of Hart and system hardware and firmware aimed at managing access to shared physical resources across workloads, minimizing contention. For example:
+
+* Memory bandwidth management
+* Cache allocation policies across workloads, including workload prioritization
+* Hart allocation policies across workloads
+
+These types of features rely on measuring resource utilization of workloads, similar to event counters, and optimizing resource allocation policies.
+
+[width=100%]
+[%header, cols="5,20"]
+|===
+| ID#
+| Requirement
+
+| CAT_NNN
+| Lower privilege software MUST NOT be able to measure higher privilege software.
+
+| CAT_NNN
+| Software in one domains MUST NOT be able to measure software in a different domain, without consent.
+
 |===
 
+==== Denial of service
+
+The RISC-V security model is primarily concerned with protection of assets. 
+
+_Denial of service (DoS)_ is out of scope of this specification. For example, a hosting environment is free to apply their own resource allocation policy to any workloads. Including denying service. This applies in the same way to confidential workloads.
+
```

Comment:
I think there may still want to be a statement that lower privileged SW should not be able to cause DoS on the resource manager - even if the resource manager is outside the TCB. For example, while a CVM having the ability to block all interrupts may be a security concern from an infrastructure protection perspective to the service provider.

Reply:
+1 we should generally state that a "tenant" which may be a process/container/VM (ordinary VM or TVM) MUST NOT be able to launch a denial of service against the hosting platform (OS/VMM/orchestration).

Reply:
Thanks, good point. I rephrased the section and added a formal requirement.

Reply:
nit - must MUST be capitalized :-)

Reply:
Not a normal sentence it doesn't ;-)

I changed the wording.